### PR TITLE
Add web socket event on user joined

### PR DESCRIPTION
**Pull request by quickskape**
I have added the functionality to emit an event to the client when a user is added to the session.
It is currently working but I am having trouble getting a unit test to cover the socket emit event being received by the client.

If anyone can take a look at this and perhaps we can start a conversation on how to test it. :smile:

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1,21 +1,29 @@
 let express = require('express')
```
Comment (tsangste) [inline app.js:1]: change all let to const

File: bin/www [hunk 2]
```diff
@@ -4,9 +4,8 @@
  * Module dependencies.
  */
 
-let app = require('../app')
+let app = require('../app').app
```
Comment (tsangste) [inline bin/www:7]: const instead of let

File: app.js (hunk 1 above)
Comment (ReissJarvis) [reply app.js:1]: Yea I agree with this! Immutability! Reading more about it I think it's better to use const over let

**Comment by ReissJarvis** [comment]
LGTM

**Pull request closed without merging**